docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,17 @@
+--
+title: Running ASP.NET MVC applications in Docker
+description: Running ASP.NET MVC applications in Docker
```

Comment:
let's try to apply the right metadata to new topics. It's ok for now since they're still placeholders.